### PR TITLE
Gun kits don't need cable coil or tools, halved crafting time

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Crafting R&D guns from gun kits no longer requires tools or cable coil. The decloner and energy crossbow still need reagents.

Halved R&D gun crafting time. 20->10 seconds.

## Why It's Good For The Game

These changes were made a long, long while ago and honestly while I understand gun kits I don't understand why it was made So. Annoying. To make the fucking guns once you got everything ready. It makes it a total annoyance. You spent 40 minutes getting all the tech for it, you shouldn't have to also get tools and cables and wait 20 seconds standing still.

Anyone who has played ingame like any time after that change can attest how underused any R&D gun is now. X-ray laser guns still DESTROY blobs but people don't even THINK about them because of the dumb annoying recipe (alongside RnD being a pain now).

Simply put this just. Makes life easier for security officers. And reduces tool dependency.

## Changelog

:cl:
qol: Crafting R&D guns from gun kits no longer requires tools or cable coil. The decloner and energy crossbow still need reagents.
qol: Halved R&D gun crafting time. 20->10 seconds.
/:cl:

### CODE REVIEW

**Comment by Serxule** [comment]
i mean, i guess you could say that the gun kit comes with everything you need, but also it makes sense that you'd need tools and cables if it was just the parts

**Comment by necromanceranne** [comment]
Looks good to me. I made them the way I did because at the time, I thought I had to include some extra items to have it make sense. Also, the tools and cable were always in the brig roundstart.

But if that's somehow presenting an issue this hopefully makes it a bit easier overall.

**Review by Ghommie — APPROVED** [review]

**Comment by necromanceranne** [comment]
It's a bit of that? Here is the underlying issue.

Mechs are science content, so understandably mechs can be in the department meant to research them. Roboticists have mechs as part of their job content, so mech building is a role draw. It's all built in robotics.

Security have a much broader array of duties, no direct control over weapons research, and aren't directly informed of its completion. In addition, you need a laser gun or energy gun, which requires a bit of effort to get for normal officers if the armory isn't just opened. The tech is on the way to mech weapons, but seccies aren't necessarily driven to go get themselves weapon upgrades.

I'll be dead and buried before we ever see weapon printing return, but even so much as printing weapon kits seems like it has an inherent flaw that need some rethinking. And anyone willing to improve the situation is very much welcome.

**Comment by Vincent983** [comment]
also there's the fact that most of the advanced weapons are either situational or downgrades, like the advanced energy gun being self-recharging, but it takes ages and you aren't able to stick it in a weapon recharger, x-ray lasers existing to fuck over milfs and blobs, and the full-sized energy crossbow needing... a protokinetic accelerator for some reason

**Comment by SkeletalElite** [comment]
> Security have a much broader array of duties, no direct control over weapons research,

well technically the HoS has the research app on his PDA and can research the guns if he wants but I think most HoS players don't realize that

**Comment by carlarctg** [comment]
@Ghommie is this *really* balance? The only big difference is not needing tools.

**Comment by Ghommie** [comment]
Fine, then let's say it's both, since it also makes 'life' easier as you say.

**Comment by carlarctg** [comment]
fucked!

**Comment by Loafin34** [comment]
> also there's the fact that most of the advanced weapons are either situational or downgrades, like the advanced energy gun being self-recharging, but it takes ages and you aren't able to stick it in a weapon recharger, x-ray lasers existing to fuck over milfs and blobs but having lower damage otherwise, and the full-sized energy crossbow needing... a protokinetic accelerator for some reason

can put it in recharger
